Makefile

```diff
@@ -36,12 +36,12 @@ build: build-non-appimage
 build-non-appimage: clean
 	tar -czf module.tar.gz run.sh requirements.txt src
 
-build-appimage: clean
+build-appimage-aarch64: clean
 	docker build -t $(IMAGE_NAME) . && \
 	docker run --name $(CONTAINER_NAME) $(IMAGE_NAME) && \
-	docker cp $(CONTAINER_NAME):/app/$(APPIMAGE_NAME) ./$(APPIMAGE_NAME) && \
-	chmod +x ./${APPIMAGE_NAME} && \
-	tar -czf module.tar.gz run.sh $(APPIMAGE_NAME)
+	docker cp $(CONTAINER_NAME):/app/$(AARCH64_APPIMAGE_NAME) ./$(AARCH64_APPIMAGE_NAME) && \
+	chmod +x ./${AARCH64_APPIMAGE_NAME} && \
+	tar -czf module.tar.gz run.sh $(AARCH64_APPIMAGE_NAME)
```

Comment:
1. might want to `docker rm` the container at the end of this (otherwise it will fail to run the second time you do this, I suspect)
2. fairly sure you don't need the `&& \` at the end of the lines -- make will exit if any line fails

Reply:
1. I think clean at the beginning takes care of this. Is it better to do it at the end instead of the beginning though? I remove at the beginning because during development I would look at the file system of the container.
2. Got it

Reply:
oops didn't notice clean. yup clean solves this